lib/zlib.js

```diff
@@ -71,6 +71,7 @@ const {
 const { owner_symbol } = require('internal/async_hooks').symbols;
 const {
   validateFunction,
+  validateNumber
```

Comment:
nit: trailing comma

Reply:
OK. But the lint seems passed. Shall we add this to the lint rule?

Reply:
> OK. But the lint seems passed. Shall we add this to the lint rule?

It would be nice if the lint rule could be added